firmware/quintuna/CRIT/src/io/io_canMsgQueues.h

```diff
@@ -13,6 +13,19 @@ namespace io
 {
 namespace can1
 {
+    void init(){
+        hw::can::can1.init();
+    }
+
+    void CanMsgInit(){
+        hw::can::CanMsg tx_msg{};
```

Comment:
implementations for these functions should be in their own `.cpp` file. We put data definitions/function declarations in headers, and everything else in the source files